src/bin/pgcopydb/indexes.c

```diff
@@ -1132,10 +1130,10 @@ copydb_create_constraints(CopyDataSpec *specs, PGSQL *dst, SourceTable *table)
 		 * When running pgcopydb create constraints, that information is more
 		 * relevant.
 		 */
-		int logLevel =
+		int level =
```

Comment:
Why that change? Is `cppcheck` opinionated about variable names?

The strategy for variable names in pgcopydb comes from the one picked by the Citus team, which is heavily inspired by the book [Code Complete, Second Edition](https://www.oreilly.com/library/view/code-complete-2nd/0735619670/). We are taking more liberties in pgcopydb because of personal opinions, but still, if you're looking for a documented code style, that would probably be one that's relevant.

Reply:
this is a fix for the `shadowed variable` warning which means there is another variable/function with the same name in the outer scope.